src/resources/js/admin/help-hub-iframe.js

```diff
@@ -91,15 +91,15 @@ window.DocsBotAI = window.DocsBotAI || {};
 	 * @return {void}
 	 */
 	obj.setup = () => {
-		const isOptedIn = obj.selectors.body.getAttribute( 'data-opted-in' ) === '1';
+		const isOptedIn = document.querySelector( obj.selectors.body ).getAttribute( 'data-opted-in' ) === '1';
```

Comment:
Same as below - put the object in a const and reference it rather than doing a the query multiple times.